CONTRIBUTING.md

```diff
@@ -48,12 +48,12 @@ To make the contribution process as seamless as possible, we ask for the followi
   * Sign the [CLA](https://www.mongodb.com/legal/contributor-agreement).
   * Have test cases for the new code. If you have questions about how to do this, please ask in your pull request or check the [Building and Testing](#building-and-testing) section.
   * Run `make fmt`.
-  * Add documentation if you are adding new features or changing functionality.  
+  * Add documentation if you are adding new features or changing functionality.
```

Comment:
Change to: `Add documentation annotations`

Reply:
Could we keep this as is and leave the annotation mentioned in the requirements section? I'm afraid we might generate confusion between docs and the actual `Annotations` structure.